test/status-test.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const test = require('tape');
+const CircuitBreaker = require('../');
+const Status = require('../lib/status.js');
+const { passFail } = require('./common');
+
+test('CircuitBreaker status - new Status static method', t => {
+  t.plan(9);
+
+  const prevStats = {
+    failures: 1,
+    fallbacks: 1,
+    successes: 1,
+    rejects: 1,
+    fires: 1,
+    timeouts: 1,
+    cacheHits: 1,
+    cacheMisses: 1,
+    semaphoreRejections: 1,
+    percentiles: {},
+    latencyTimes: []
+  };
+
+  const status = CircuitBreaker.newStatus({stats: prevStats});
+  t.equal(status instanceof Status, true, 'returns a new Status instance');
+
+  const stats = status.stats;
+  t.equal(stats.failures, 1, 'status reports 1 failure');
+  t.equal(stats.rejects, 1, 'status reports 1 reject');
+  t.equal(stats.fires, 1, 'status reports 5 fires');
```

Comment:
status reports 5 fires, should be 1 fire?

Reply:
yea,  i can't copy/paste very good

Reply:
Updated